Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -89,8 +89,10 @@ TerminalWidget::TerminalWidget(int ptm_fd, bool automatic_size_policy)
     : m_terminal(*this)
     , m_automatic_size_policy(automatic_size_policy)
 {
-    static_assert(sizeof(m_colors) == sizeof(xterm_colors));
-    memcpy(m_colors, xterm_colors, sizeof(m_colors));
+    VERIFY(m_colors.size() == xterm_colors.size());
+    for (size_t i = 0; i < xterm_colors.size(); i += 1) {
```

Comment:
Can you use `i++` instead of `i += 1` here? (They do the same thing here, but the former is more idiomatic in three-part for loops.)

Reply:
Done.